Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Specify the base image
+FROM node:14
```

Comment:
Can we bump this to 20 since it's LTS now?

Reply:
yep!

Reply:
Should i just make it `node:lts`? Or do you prefer a pinned version

Reply:
I was too slow - I prefer the pin haha. Also I think we can drop node-fetch and use native fetch after 18+ :)